packages/atomic/src/components/search/tabs/atomic-tab-manager/atomic-tab-manager.tsx

```diff
@@ -5,14 +5,11 @@ import {
   Tab,
   buildTab,
```

Comment:
Unrelated to this PR but while looking at the file I see that the tab-manager is tagged as  `@alpha`. When will it be public ?

Reply:
@louis-bompart what's the plan for this?